### PR TITLE
Update UG and README

### LINKED ISSUE #460: Update UG and Readme

### PR DESCRIPTION
**Pull request by johannagwan**
Closes #460

Changes include:
1. README
- Remove `export to PDF` part
2. UG
- Command Legend: remove `BODY` part and make necessary changes
- Paths: improve grammar, add `symbols` to title naming restrictions, fix typo in `RelativePath`
- New: remove body, add `symbols` to title naming restrictions, TODO: update figma diagram to remove the body (can someone help do this?)
- Edit: change "replace" to "update", change `Format`, will leave the more detailed changes for section 3.5.1 to @HemanshuGandhi (as stated in #380 )
- Search: remove "coming in v2.0", move up to 3.7
- Sug: change `TAB` to `ENTER`, add `coming in v2.0` for paths that include `../`
- Command Summary: update `new` and `edit` format

Can someone help change in all the figma diagrams, from "Press tab to autofill" to "Press enter to autofill"? @HemanshuGandhi @ljiazh3ng @firzanarmani @kevinputera

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -49,27 +49,27 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 *Command Legend*
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `open NAME`, `NAME` is a parameter which can be used as `open Notably`.
-* Items in square brackets are optional e.g `-t TITLE [-b BODY]` can be used as `-t Notably -b Lorem ipsum` or as `-t Notably`.
-* Parameters can be in any order e.g. if the command specifies `-t TITLE -b BODY`, `-b BODY -t TITLE` is also acceptable.
+* Items in square brackets are optional e.g both `open -t Notably` and `open Notably` will open the note titled "Notably".
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: Hmmm I feel like we need to show the user what is the notation (what we mean by square brackets). For this case, we should show the user that the command is `open [-t] TITLE`

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -202,19 +194,19 @@ image::demo/Open.png[]
 
 === Edit an existing note: `edit`
 
-If you want to change the `BODY` content of a note, use the `edit` command. This command *replaces* the existing `BODY` content of the currently open note, with the new `BODY` content. +
+If you want to change the `BODY` content of a note, use the `edit` command. This command *updates* the existing `BODY` content of the currently open note, with the new `BODY` content. +
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: It's not really accurate to say that this command updates the existing body. Rather, I was thinking that we could phrase it this way:

"If you want to change the `BODY` content of the currently opened note, use the `edit` command. This command *opens* an editing window where you can update the existing `BODY` of the note.

What do you think?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -202,19 +194,19 @@ image::demo/Open.png[]
...
 ====
 * You are not allowed to `edit` the root note, i.e the currently opened path is ``"/"``
 ====
 
 ==== Example: Editing the currently opened note
-. Let's replace the `BODY` content of the currently opened note `Workspace`, with "This is the new paragraph". This can be done with the command: +
+. Let's update the `BODY` content of the currently opened note `Workspace`, with "This is the new paragraph". This can be done with the command: +
 
-    edit -b This is the new paragraph
+    edit
```
Comment (kevinputera) [inline docs/UserGuide.adoc:208]: OK, we should update this (since we're now using a modal), hahaha. I'll take note of this.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -297,9 +293,9 @@ image::demo/Search.png[]
 
 * *View help*: `help`
 * *Exit the program*: `exit`
-* *Create a new note*: `new -t TITLE [-b BODY] [-o]`
+* *Create a new note*: `new -t TITLE [-o]`
 * *Open an existing note*: `open [-t] [PATH/TO/]TITLE`
-* *Edit an existing note*: `edit -b BODY`
+* *Edit an existing note*: `edit`
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: Should we rephrase this as "Opens an editing window"?

**Review by kevinputera — APPROVED** [review]
@johannagwan thanks for the initiative! I have left some comments, what do you think? Otherwise, LTGM!

File: docs/UserGuide.adoc (hunk 1 above)
Comment (johannagwan) [reply docs/UserGuide.adoc]: Will rephrase!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (johannagwan) [reply docs/UserGuide.adoc]: Alright, will change!

File: docs/UserGuide.adoc (hunk 4 above)
Comment (johannagwan) [reply docs/UserGuide.adoc]: Ok!

File: docs/UserGuide.adoc (hunk 3 above)
Comment (johannagwan) [reply docs/UserGuide.adoc:208]: Ok, will leave it to you!
